app/controllers/vm_infra_controller.rb

```diff
@@ -73,4 +73,16 @@ def skip_breadcrumb?
 
   menu_section :inf
   has_custom_buttons
+
+  def vm_transform
+    vm = Vm.find_by(:id => params[:id].to_i)
+    @right_cell_text = _("Transform VM %{name} to RHV") % {:name => vm.name}
+    dialog = Dialog.find_by(:label => 'Transform VM')
```

Comment:
Huh.. finding a dialog by label doesn't sound quite that stable, any chance you can find it by something else?

And, probably should be using RBAC here too, somehow.